### PR TITLE
[DAT-1273] - Add contact information page when the user has transfer as payment method

### PR DESCRIPTION
**Pull request by jhoffman-ms**
Add contact information page when the user has transfer as payment method.

image

**Task:** [DAT-1273](https://makingsense.atlassian.net/jira/software/projects/DAT/boards/62?selectedIssue=DAT-1273)

### CODE REVIEW

File: src/components/Plans/UpdatePaymentInformation/UpdatePaymentInformation/ContactInformation/index.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+const fieldNames = {
+  firstname: 'firstname',
+  lastname: 'lastname',
+  email: 'email',
+  phone: 'phone',
+};
+
+const displayBlockStyle = {
+  display: 'block',
+};
```
[the remark this answers is not in the document]
Comment (jhoffman-ms) [reply src/components/Plans/UpdatePaymentInformation/UpdatePaymentInformation/ContactInformation/index.js:26]: @jcamposmk , how should it be done?

File: src/services/doppler-billing-user-api-client.ts [hunk 2]
```diff
@@ -497,4 +507,25 @@ export class HttpDopplerBillingUserApiClient implements DopplerBillingUserApiCli
...
+        method: 'POST',
+        url: `/accounts/${email}/payments/reprocess/send-contact-information-notification`,
+        data: this.mapSendContantInformation(values),
+        headers: { Authorization: `bearer ${jwtToken}` },
+      });
+
+      if (response.status === 200) {
+        return { success: true };
+      } else {
+        return { success: false };
```
Comment (jcamposmk) [inline src/services/doppler-billing-user-api-client.ts:525]:
```suggestion
return { success: false, error: response };
```

File: src/components/Plans/UpdatePaymentInformation/UpdatePaymentInformation/ContactInformation/index.js [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+      if (result.success) {
+        setStatus(SAVED);
+
+        createTimeout(() => {
+          navigate('/login');
+        }, DELAY_BEFORE_REDIRECT_TO_LOGIN);
+      } else {
+        setStatus(HAS_ERROR);
+      }
+    };
```
Comment (jcamposmk) [inline src/components/Plans/UpdatePaymentInformation/UpdatePaymentInformation/ContactInformation/index.js:56]: can you add some test for to verify when the form is submited and the user is redirect?

File: src/components/Plans/UpdatePaymentInformation/UpdatePaymentInformation/ContactInformation/index.js [hunk 4]
```diff
@@ -0,0 +1,175 @@
...
+                                    status === SAVED
+                                      ? 'updatePaymentMethod.payment_method.transfer.send_email_success_message'
+                                      : 'updatePaymentMethod.payment_method.transfer.send_email_error_message',
+                                  )}
+                                </p>
+                              </div>
+                            </div>
+                            <hr className="m-t-24 m-b-24" />
+                          </FieldItem>
+                        )}
```
Comment (jcamposmk) [inline src/components/Plans/UpdatePaymentInformation/UpdatePaymentInformation/ContactInformation/index.js:163]: What do you think about create a component with this code and test it separately?

**Pull request merged**